doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst

```diff
@@ -245,3 +254,118 @@ The following samples allow you to test if the buttons (or switches) and LEDs on
++------------------+-----------------------+----------------------------------+
+| D5               | SW_CTRL0              | Switch control 0                 |
++------------------+-----------------------+----------------------------------+
+| D6               | COEX_STATUS1          | Coexistence status 1             |
++------------------+-----------------------+----------------------------------+
+| D7               | HOST_IRQ              | Interrupt request to host        |
++------------------+-----------------------+----------------------------------+
+| D8               | DATA2                 | QSPI data line 2                 |
++------------------+-----------------------+----------------------------------+
+| D9               | DATA3                 | QSPI data line 3                 |
++------------------+-----------------------+----------------------------------+
+| D10              | SS                    | Slave select                     |
++------------------+-----------------------+----------------------------------+
+| D11              | MISO/DATA1            | QSPI/SPI Data line 1/ Slave Out  |
++------------------+-----------------------+----------------------------------+
+| D12              | MOSI/DATA0            | QSPI/SPI Data line 0/ Slave In   |
++------------------+-----------------------+----------------------------------+
+| D13              | CLK                   | QSPI/SPI Clock                   |
++------------------+-----------------------+----------------------------------+
+| GND              | GND                   | Ground                           |
++------------------+-----------------------+----------------------------------+
+| AREF             | N.C.                  | Not used                         |
++------------------+-----------------------+----------------------------------+
+| SDA              | N.C.                  | Not used                         |
++------------------+-----------------------+----------------------------------+
+| SCL              | N.C.                  | Not used                         |
++------------------+-----------------------+----------------------------------+
+
+Programming
+-----------
+
+To add support for the nRF7002 EK on an application running on a compatible host development board, the ``SHIELD`` setting must be specified.
+
+The nRF7002 EK features the nRF7002 companion IC.
+In addition, the shield may be used to emulate the nRF7001 and nRF7000 companion IC variants
+
+To add support for the nRF7002 EK and the nRF7002 IC, set ``-DSHIELD=nrf7002ek`` when you invoke ``west build`` or ``cmake`` in your |NCS| application.
+To emulate support for the nRF7001 or nRF7000 ICs, specify ``-DSHIELD=nrf7002ek_nrf7001`` or ``-DSHIELD=nrf7002ek_nrf7000``, respectively.
+
+Alternatively, add the shield in the project's :file:`CMakeLists.txt` file, specifying the below settings, depending on which IC is to be used:
+
+.. code-block:: console
```

Comment:
would it be better looking if we merged these code blocks in one, with multiple lines?

Reply:
These commands are referring to individual ICs, hence separate code blocks.
Generally, a series of following commands are listed in the same code block.
But, if separate code blocks are not serving the purpose, I will change.